airflow/contrib/kubernetes/kubernetes_request_factory/pod_request_factory.py

```diff
@@ -75,6 +77,7 @@ class ExtractXcomPodRequestFactory(KubernetesRequestFactory):
     _yaml = """apiVersion: v1
 kind: Pod
 metadata:
+  namespace: default
```

Comment:
does that give "default" or "dev" as default namespace?

Reply:
default